ci/scripts/check-npm-dependencies.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable camelcase */
+
+const fs = require('fs');
+const path = require('path');
+const child_process = require('child_process');
+
+/**
+ * node check-npm-dependencies.js <package_name>
+ */
+
+const packages = fs.readdirSync(path.join(__dirname, '..', '..', 'packages'), {
+    encoding: 'utf-8',
+});
+
+const args = process.argv.slice(2);
+
+if (args.length < 1) throw new Error('Check npm dependencies requires 1 parameter: package name');
+const [packageName] = args;
+
+if (!packages.includes(packageName)) {
+    throw new Error(`provided package name: ${packageName} must be one of ${packages}`);
+}
+
+const ROOT = path.join(__dirname, '..', '..');
+
+const result = [];
+
+const checkPackageDependencies = packageName => {
+    const rawPackageJSON = fs.readFileSync(
+        path.join(ROOT, 'packages', packageName, 'package.json'),
+    );
+
+    const packageJSON = JSON.parse(rawPackageJSON);
+    const { dependencies } = packageJSON;
+
+    Object.entries(dependencies).forEach(([dependency, version]) => {
+        // is not a dependency released from monorepo. we don't care
+        if (!dependency.startsWith('@trezor')) {
```

Comment:
this could be broken by this PR https://github.com/trezor/trezor-suite/pull/5647/files#diff-477e81d5f4c3ac953698a0433c784cc0ed39a3bf532b79b2442e80c5d611b207R2

It could be fixed easily on any side, just a note to keep in mind